application/audit/services/chapter_repair_service.py

```
@@ -0,0 +1,274 @@
+
+        return Prompt(system=system, user="\n".join(user_parts))
+
+    def _schedule_aftermath(self, novel_id: str, chapter_number: int, content: str) -> None:
+        """异步触发章后管线"""
+        if not self._aftermath or not content.strip():
+            return
+
+        async def _run() -> None:
+            try:
+                await self._aftermath.run_after_chapter_saved(novel_id, chapter_number, content)
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)